dev/com.ibm.ws.jndi/src/com/ibm/ws/jndi/internal/WASURLObjectFactoryFinder.java

```diff
@@ -54,16 +53,26 @@ public ObjectFactory findFactory(String urlSchema, Hashtable<?, ?> env) throws N
         // Always add default prefix
         pkgPrefixes.add("com.sun.jndi.url");
 
+        try {
+            return findFactory(urlSchema, pkgPrefixes);
+        } catch (ClassNotFoundException e) {
+            // auto-ffdc
+        }
+    }
```

Comment:
When I build locally with Java11, it made me add a return on findFactory (I added a return null at the end).

Reply:
Thanks — an issue with my IDE prevented it flagging some errors. I found them in the build and resolved them.